### DISCUSSION TITLE
how can i import my own .py file and use this in the script node?

### DISCUSSION THREAD

**Discussion by luckychris** [JacquesLucke/animation_nodes#1918] [Q&A] [answered]
i have written a function in another text (.py) file and want to import this in a .py file which i want to use in a script node.

So basically my script .py file looks like this:

```
import driverFunctions

return driverFunctions.getTargetRelativeToCar(wantVector=True)
```
But i got the error:

<img width="209" alt="image" src="https://github.com/JacquesLucke/animation_nodes/assets/232549/bcc7b39a-0f92-4718-8630-f043210f9b15">

how can i realize this?

**Comment by OmarEmaraDev** [comment, marked as answer]
I think the Blender way to do that would be:

```
driverFunctions = bpy.data.texts['driverFunctions.py'].as_module()
```

**Reply by luckychris** [reply]
Thank you very much Omar, i didn't know that method at all and it worked like a charm!!!
